mozetl/hardware_report/hardware_dashboard.py

```diff
@@ -0,0 +1,50 @@
+import click
+import glob
+from mozetl.hardware_report.summarize_json import *
+from click_datetime import Datetime
+from datetime import datetime, timedelta
+
+
+@click.command()
+@click.option('--start_date',
+              type=Datetime(format='%Y%m%d'),
+              required=True,
+              help='Start date, only use this when backfilling (e.g. yyyy/mm/dd)')
+@click.option('--end_date',
+              type=Datetime(format='%Y%m%d'),
+              default=None,
+              help='End date, only use this when backfilling (e.g. yyyy/mm/dd)')
+@click.option('--bucket',
+              required=True,
+              help='S3 Public Bucket')
+def main(start_date, end_date, bucket):
+    if start_date:
+        end_date = start_date + timedelta(days=7)
```

Comment:
If `end_date` is specified, it should use the given value. If it is not specified, then use `start_date + timedelta(days=7)`.